packages/canary-react/src/stories/ic-time-input.stories.js

```diff
@@ -0,0 +1,435 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { IcButton, IcLink, IcTypography } from "@ukic/react";
+import React, { useState } from "react";
+import readme from "../../../canary-web-components/src/components/ic-time-input/readme.md";
+import { IcTimeInput } from "../components";
+
+export default {
+  title: "React Components/Time Input",
+  component: IcTimeInput,
+  parameters: {
+    componentAPI: {
+      data: readme,
+    },
+  },
+};
+
+/**
+ * Use the time input component to manually type in a time in different formats.
+ *
+ * The input `label` is the only required attribute for time input.
+ *
+ * Click the 'Component API' tab to view all the available attributes, events and slots for time input.
+ *
+ * To use the time input component, import `@ukic/canary-web-components` into your application.
+ *
+ * Each example below demonstrates the time input in different time formats:
+ * - HH:MM:SS (Default time format)
```

Comment:
Sorry, these two lines in the React storybook need removing as well - happy to approve after this!